e2e/benchmarks/local-benchmark/tflite_worker.js

```diff
@@ -15,80 +15,64 @@
+      inputs: model.inputs,
+      getProfilingResults() {
+        return model.getProfilingResults();
+      },
+      cleanUp() {
+        model.modelRunner.cleanUp();
+      },
+      async predict(inputData) {
         const inputTensorArray = [];
         let outputTensor;
-        try {
-          let inputData = message.data.inputData;
-          if (!inputData[0].length) {
-            // Single input, move it into an arrary
-            inputData = [inputData];
-          }
-          for (let i = 0; i< inputs.length; i++) {
-            const inputTensor = tf.tensor(
-                inputData[i], inputs[i].shape, inputs[i].dtype);
-            inputTensorArray.push(inputTensor);
-          }
-          outputTensor = tfliteModel.predict(inputTensorArray);
-          if (tfliteModel.outputs.length > 1) {
-            // Multiple outputs
-            let outputData = [];
-            for (let tensorName in outputTensor) {
-              outputData.push(outputTensor[tensorName].dataSync());
-            }
-          } else {
-            // Single output
-            const outputData = outputTensor.dataSync();
+        if (!inputData[0].length) {
+          // Single input, move it into an arrary
+          inputData = [inputData];
+        }
+        for (let i = 0; i< this.inputs.length; i++) {
+          const inputTensor = tf.tensor(
+              inputData[i], this.inputs[i].shape, this.inputs[i].dtype);
+          inputTensorArray.push(inputTensor);
+        }
+        outputTensor = model.predict(inputTensorArray);
+        if (model.outputs.length > 1) {
+          // Multiple outputs
+          let outputData = [];
+          for (let tensorName in outputTensor) {
+            outputData.push(outputTensor[tensorName].dataSync());
           }
-          // We encourage user processing output data in worker thread
-          // rather than posting output data to main thread directly, as
-          // which would bring much overhead if the output size is huge.
-          // From this perspective, we don't post output data to main thread
-          // in this benchmark.
-          postMessage('OK');
-        } catch(e) {
-          postMessage({ error: e.message });
-        } finally {
-          // dispose input and output tensors
-          tf.dispose(inputTensorArray);
-          tf.dispose(outputTensor);
+        } else {
+          // Single output
+          const outputData = outputTensor.dataSync();
         }
-        break;
-      default:
-        break;
-    }
-  }
+        // dispose input and output tensors
+        tf.dispose(inputTensorArray);
+        tf.dispose(outputTensor);
+        // We encourage user processing output data in worker thread
+        // rather than posting output data to main thread directly, as
+        // which would bring much overhead if the output size is huge.
+        // From this perspective, we don't post output data to main thread
+        // in this benchmark.
```

Comment:
Nit
```suggestion
        // We encourage the user to process output data in the worker thread
        // rather than posting output data to main thread directly, as
        // this would bring much overhead if the output size is huge.
        // From this perspective, we don't post output data to main thread
        // in this benchmark.
```

Reply:
Thanks, done.